### DISCUSSION TITLE
Custom weapons

### DISCUSSION THREAD

**Discussion by AngrySkype** [nzp-team/nzportable#302] [General]
I'd just like to know if it's possible to make custom weapons for the game, and if so, how, or if not, when?

**Comment by BCDeshiG** [comment]
Currently there's no way of doing custom weapons without modifying the game itself to my knowledge. There was an old branch that aimed to have data-driven weapons but that hasn't been touched in a while.

**Reply by AngrySkype** [reply]
When will there be support for custom weapons in this branch?

**Reply by BCDeshiG** [reply]
Realistically not any time soon seeing as how the main dev stepped down from development.

**Reply by ToonLink0001** [reply]
dam i was wondering the same thing
